# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/motor/no-deprecated.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./parser/src/parser.c.orig	2004-06-12 13:10:03.228842992 +0300
+++ ./parser/src/parser.c	2004-06-12 13:09:46.551378352 +0300
@@ -313,7 +313,7 @@
 		buf = mmap(0, *size, PROT_READ, MAP_SHARED, f, 0);
 		if (buf == MAP_FAILED) buf = NULL;
 		break;
-	    default:
+	    default: ;
 	}
     }
     close(f);
@@ -333,7 +333,7 @@
 	    // munmap
 	    munmap(text, size);
 	    break;
-	default:
+	default: ;
     }
 }
 
